DESCRIPTION

```diff
@@ -4,24 +4,40 @@ Description: Programmatic interface to the
     Web Service methods provided by the National 'Phenology' Network
     (<https://usanpn.org/>), which includes data on various life history
     events that occur at specific times.
-Version: 0.1.4.9100
+Version: 1.1.1
+Date: 2020-10-27
 License: MIT + file LICENSE
 Authors@R: c(
-    person("Scott", "Chamberlain", role = c("aut", "cre"), email = "myrmecocystus@gmail.com"),
-    person("Lee", "Marsh", role = "aut", email = "npnlee85@gmail.com")
+    person("Lee", "Marsh", role = c("aut", "cre"), email = "lee@usanpn.org"),
+    person("Scott", "Chamberlain", role = "aut", email = "myrmecocystus@gmail.com"),
```

Comment:
you can remove my email address

Reply:
Just your email or your name on the whole? You've been a guiding force and a mentor, seems appropriate to give credit where it's due!

Reply:
just remove the email address, keep my name :)